posthog/session_recordings/sql/session_replay_event_sql.py

```diff
@@ -130,8 +130,37 @@ def KAFKA_SESSION_REPLAY_EVENTS_TABLE_SQL(on_cluster=True):
     )
 
 
-def SESSION_REPLAY_EVENTS_TABLE_MV_SQL(on_cluster=True):
-    return """
+def SESSION_REPLAY_EVENTS_TABLE_MV_SQL(on_cluster=True, exclude_columns=None):
+    exclude_columns = exclude_columns or []
+
+    target_table = "writable_session_replay_events"
+    on_cluster_clause = ON_CLUSTER_CLAUSE(on_cluster)
+    database = settings.CLICKHOUSE_DATABASE
+
+    # ClickHouse is incorrectly expanding the type of the snapshot source column
+    # Despite it being a LowCardinality(Nullable(String)) in writable_session_replay_events
+    # The column expansion picks only Nullable(String) and so we can't select it
```

Comment:
ah... would be so cool to defuckulate that lowsnapshot nullable string mess 🙈

Reply:
(not blocking here! just thinking out loud - at some point i notice i've thought the thing multiple times as the signal i should do it 🤣 )